### PR TITLE
actually memoffset also had an uninit-drop vuln

### PR DESCRIPTION
**Pull request by RalfJung**
This affects all versions ever published.

### CODE REVIEW

File: crates/memoffset/RUSTSEC-2019-0011.toml [hunk 1]
```diff
@@ -2,13 +2,13 @@
 id = "RUSTSEC-2019-0011"
 package = "memoffset"
 date = "2019-07-16"
-title = "Flaw in offset_of and span_of causes SIGILL, potential memory unsafety"
+title = "Flaw in offset_of and span_of causes SIGILL, drops uninitialized memory of arbitrary type on panic in client code"
 description = """
 Affected versions of this crate caused traps and/or memory unsafety by zero-initializing references.
+They also could leat to uninitialized memory being dropped if the field for which the offset is requested was behind a deref coercion, and that deref coercion caused a panic.
```
Comment (Shnatsel) [inline crates/memoffset/RUSTSEC-2019-0011.toml]: s/leat/lead/

Comment (RalfJung) [reply crates/memoffset/RUSTSEC-2019-0011.toml]: Thanks, fixed.

**Comment by tarcieri** [comment]
Should this note the potential implications are memory disclosure and/or RCE?

**Comment by RalfJung** [comment]
I don't have any experience with vulnerability reports -- no idea.^^

There are no known cases of this actually happening in practice.

**Comment by tarcieri** [comment]
@RalfJung ok, will merge this as-is and we can potentially follow up on severity information

File: crates/memoffset/RUSTSEC-2019-0011.toml (hunk 1 above)
Comment (kud1ing) [inline crates/memoffset/RUSTSEC-2019-0011.toml]: "leat" => "lead"?

**Pull request merged**